.github/workflows/performance_tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Performance Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron:  '0 9 * * 0'
+
+jobs:
+  check_branch_changes:
+    runs-on: ubuntu-22.04
+    outputs:
+      changed: ${{ steps.branch_change_output.outputs.has-new-commits }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: check branch changes
+        id: branch_change_output
+        uses: adriangl/check-new-commits-action@v1
+        with:
+          seconds: 604800
+          branch: 'dev'
+
+  run_performance_tests:
+    runs-on: ubuntu-22.04
+    needs: check_branch_changes
+    if: ${{ needs.check_branch_changes.outputs.changed == 'true' }}
+    defaults:
+      run:
+        working-directory: ./test/performance
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: 'main'
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.1'
+      - run: bundle
+      - run: script/runner -B
+      - uses: actions/checkout@v2
+      - run: script/runner -C > performance_tests.txt
+      - name: Save performance results
+        uses: actions/upload-artifact@v3
+        with:
+          name: performance-test-results
+          path: test/performance/performance_tests.txt
+      - name: Slack results
+        uses: adrey/slack-file-upload-action@master
```

Comment:
Did adrey not produce a v1 release for this action? I'd rather not blindly grab the latest raw git code if possible.

Reply:
Great call out. Addressed [here](2b4f22826184fa07d9f9b7e12146bf4f780359b1).